_data-prepper/pipelines/configuration/sources/otel-logs-source.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: otel_logs_source
+parent: Sources
+grand_parent: Pipelines
+nav_order: 25
+---
+
+# otel_logs_source
+
+## Overview
+
+`otel_logs_source` is an OpenTelemetry source that follows the [OpenTelemtry Protocol Specification](https://github.com/open-telemetry/oteps/blob/master/text/0035-opentelemetry-protocol.md), receives logs from OpenTelemetry Collector, and receives `ExportLogsServiceRequest` records to process through a Data Prepper pipeline.
```

Comment:
It might be better to reword this line:

> receives logs from OpenTelemetry Collector, and receives `ExportLogsServiceRequest` records 

to:

> receives logs from the OpenTelemetry Collector in the form of `ExportLogsServiceRequest` records.

The current wording makes it seem like it can read two different things (logs and export records). But they are the same thing.